src/main/scala/eisner/eisner.scala

```diff
@@ -9,25 +9,36 @@ package object eisner {
   private[eisner] final val Topic    = dot.Topic
   private[eisner] final val Store    = dot.Store
 
+  private[this] final val decimalsPattern = """\.(\d{2})\d+(\D?)""".r
+  private[this] object trimDoubles extends shapeless.poly.->((s: String) => decimalsPattern.replaceAllIn(s, ".$1$2"))
+  private[this] final def simplify(svg: SVG): SVG = {
+    val simplified = shapeless.everywhere(trimDoubles)(svg)
```

Comment:
why is this stored to a val ?

Reply:
```
sbt:sbt-eisner> test
[info] Compiling 13 Scala sources to /Users/jsirocchi/code-personal/eisner/target/scala-2.12/sbt-1.0/classes ...
[error] /Users/jsirocchi/code-personal/eisner/src/main/scala/eisner/eisner.scala:14:87: type mismatch;
[error]  found   : svg.type (with underlying type eisner.package.SVG)
[error]  required: shapeless.poly.Case[_1.type,shapeless.HNil]{type Result = ?} where val _1: shapeless.EverywhereAux[eisner.package.trimDoubles.type]
[error]   private[this] final def simplify(svg: SVG): SVG = shapeless.everywhere(trimDoubles)(svg)
[error]                                                                                       ^
[error] one error found
```
some inference issue? see https://stackoverflow.com/questions/36725079/strange-type-error-after-inlining-a-val